sqrl-tools/sqrl-packager/src/main/java/com/datasqrl/packager/repository/RemoteRepositoryImplementation.java

```diff
@@ -30,14 +22,14 @@
 import java.net.http.HttpResponse.BodyHandlers;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Map;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
 import java.util.Optional;
+import lombok.Setter;
+import net.lingala.zip4j.ZipFile;
+import org.apache.commons.io.FileUtils;
 
 public class RemoteRepositoryImplementation implements Repository {
-  public static final String PKG_NAME_KEY = "pkgName";
-  public static final String VERSION_KEY = "version";
-  public static final String VARIANT_KEY = "variant";
-
   public static final URI DEFAULT_URI = URI.create("https://repo.sqrl.run");
```

Comment:
Currently we need to manually change it to the staging url to make this work. What should we do here? Are we going to forward this url to the new repo once we release the new version?